.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: rust
+sudo: false
+
+matrix:
+  include:
+
+  # tests pass
+  - rust: stable
+    script:
+    - cargo test --locked
```

Comment:
FYI I tend to add `--all` in case this ever becomes a workspace and I forget about updating this